exercises/java/src/main/java/com/adaptionsoft/games/trivia/runner/GameRunner.java

```diff
@@ -1,31 +1,33 @@
 
 package com.adaptionsoft.games.trivia.runner;
-import java.util.Random;
-
 import com.adaptionsoft.games.uglytrivia.Game;
+import com.adaptionsoft.games.uglytrivia.Player;
+
+import java.util.Random;
 
 
 public class GameRunner {
 
 	private static boolean notAWinner;
 
 	public static void main(String[] args) {
-		Game aGame = new Game();
-		
-		aGame.add("Chet");
-		aGame.add("Pat");
-		aGame.add("Sue");
+		Game game = new Game();
+
+		game.addPlayer(new Player("Chet"));

```

Comment:
Like the removal of the Player - feels like the right move, though maybe done as too big a change all in one.